jetty-osgi/jetty-osgi-boot/pom.xml

```diff
@@ -81,7 +81,8 @@
                 javax.transaction.xa;version="1.1.0";resolution:=optional,
                 org.objectweb.asm;version="$(version;=;${asm.version})";resolution:=optional,
                 org.osgi.framework,
-                org.osgi.service.cm;version="1.2.0",
+                org.osgi.service.cm;version="1.4.0",
+                org.osgi.service.event;version="1.4.0",
```

Comment:
Why add only `org.osgi.service.event` and not the rest of the other packages in the new osgi-service jar?

Reply:
Because clearly they're not needed to function.